src/playlist-selectors.js

```diff
@@ -261,7 +275,8 @@ export const lastBandwidthSelector = function() {
   return simpleSelector(this.playlists.master,
                         this.systemBandwidth,
                         parseInt(safeGetComputedStyle(this.tech_.el(), 'width'), 10),
-                        parseInt(safeGetComputedStyle(this.tech_.el(), 'height'), 10));
+                        parseInt(safeGetComputedStyle(this.tech_.el(), 'height'), 10),
+                        this.limitRenditionByPlayerDimensions);
```

Comment:
Here and below, I don't think this value will ever be set, since `limitRenditionByPlayerDimensions` is set within the `options_` object property and not on the `HlsHandler` instance directly.

Reply:
I really should have spent some more time running the code via the debugger to make sure things were getting passed around. I did some more testing using this m3u8 which I created with [Mux per title encoding](https://docs.mux.com/docs/per-title-encoding) - https://stream.mux.com/d3dYsgwvtxk101D6ExK3HuoKe3aps9IIA.m3u8 - you can pretty clearly spot the difference on the test page when setting `limitRenditionByPlayerDimensions: false` if you use this manifest.

this is my PR for this repo so I was all like...

![](https://media3.giphy.com/media/SRx5tBBrTQOBi/giphy.gif?cid=3640f6095bb39e38476679596ffa5da4)

I'm going to update your contribution guidelines with some learnings from this process in another PR